Infinite Crash Loop

<!--
Thank you for your help in making FlorisBoard better!

Guide to a good crash-report:
• Please search existing bug/crash reports to avoid creating duplicates.
• Give your crash report a good name (no generics like "Error" or "Crash"), so others can easily identify the topic of your issue.
• Describe what you were doing what could've led to the crash and whether the crash is random or reproducible.
-->

#### Short description
Describe what you were doing that could've led to the crash.

#### Steps to reproduce
1. Go to Select Input Method
2. Click on FlorisBoard Beta
3. 
4. See crash

<!-- Paste the generated crash log below -->
#### Environment information
- FlorisBoard Version: 0.3.11-beta02 (37)
- Device: HUAWEI DUB-LX1
- Android: 8.1.0 (cn=O_MR1 sdk=27)

#### Features enabled
```kt
smartbar = true
suggestions = true
suggestions_clipboard = true
suggestions_next_word = true
glide = false
clipboard_internal = true
clipboard_history = false
```

#### Attached stacktrace files
<details>
<summary>1620059142916.stacktrace</summary>

```
android.view.InflateException: Binary XML file line #2: Binary XML file line #2: Error inflating class <unknown>
Caused by: android.view.InflateException: Binary XML file line #2: Error inflating class <unknown>
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.newInstance0(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:334)
	at android.view.LayoutInflater.createView(LayoutInflater.java:658)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:801)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
	at android.view.View.inflate(View.java:23459)
	at dev.patrickgold.florisboard.ime.media.emoji.EmojiKeyboardView.<init>(EmojiKeyboardView.kt:83)
	at dev.patrickgold.florisboard.ime.media.emoji.EmojiKeyboardView.<init>(EmojiKeyboardView.kt:65)
	at dev.patrickgold.florisboard.ime.media.emoji.EmojiKeyboardView.<init>(EmojiKeyboardView.kt:64)
	at dev.patrickgold.florisboard.ime.media.MediaInputManager.createTabViewFor(MediaInputManager.kt:164)
	at dev.patrickgold.florisboard.ime.media.MediaInputManager.access$createTabViewFor(MediaInputManager.kt:48)
	at dev.patrickgold.florisboard.ime.media.MediaInputManager$onRegisterInputView$1$4.invokeSuspend(MediaInputManager.kt:113)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at android.os.Handler.handleCallback(Handler.java:809)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:166)
	at android.app.ActivityThread.main(ActivityThread.java:7555)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:469)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:963)
Caused by: java.lang.IllegalArgumentException: The style on this component requires your app theme to be Theme.AppCompat (or a descendant).
	at com.google.android.material.internal.ThemeEnforcement.checkTheme(ThemeEnforcement.java:243)
	at com.google.android.material.internal.ThemeEnforcement.checkAppCompatTheme(ThemeEnforcement.java:213)
	at com.google.android.material.internal.ThemeEnforcement.checkCompatibleTheme(ThemeEnforcement.java:148)
	at com.google.android.material.internal.ThemeEnforcement.obtainStyledAttributes(ThemeEnforcement.java:76)
	at com.google.android.material.tabs.TabLayout.<init>(TabLayout.java:508)
	at com.google.android.material.tabs.TabLayout.<init>(TabLayout.java:488)
	... 24 more

```
</details>

--- a/app/src/main/java/dev/patrickgold/florisboard/ime/core/FlorisBoard.kt
+++ b/app/src/main/java/dev/patrickgold/florisboard/ime/core/FlorisBoard.kt
@@ -91,7 +91,7 @@ class FlorisBoard : InputMethodService(), LifecycleOwner, FlorisClipboardManager
      * service class should be used directly.
      */
     private var _themeContext: Context? = null
-    private val themeContext: Context
+    val themeContext: Context
         get() = _themeContext ?: this
 
     lateinit var prefs: PrefHelper
@@ -148,7 +148,6 @@ class FlorisBoard : InputMethodService(), LifecycleOwner, FlorisClipboardManager
     lateinit var asyncExecutor: ExecutorService
 
     companion object {
-
         @Synchronized
         fun getInstance(): FlorisBoard {
             return florisboardInstance!!

--- a/app/src/main/java/dev/patrickgold/florisboard/ime/media/MediaInputManager.kt
+++ b/app/src/main/java/dev/patrickgold/florisboard/ime/media/MediaInputManager.kt
@@ -161,9 +161,9 @@ class MediaInputManager private constructor() : CoroutineScope by MainScope(),
      */
     private fun createTabViewFor(tab: Tab): LinearLayout {
         return when (tab) {
-            Tab.EMOJI -> EmojiKeyboardView(florisboard)
-            Tab.EMOTICON -> EmoticonKeyboardView(florisboard)
-            else -> LinearLayout(florisboard).apply {
+            Tab.EMOJI -> EmojiKeyboardView(florisboard.themeContext)
+            Tab.EMOTICON -> EmoticonKeyboardView(florisboard.themeContext)
+            else -> LinearLayout(florisboard.themeContext).apply {
                 addView(TextView(context).apply {
                     text = "not yet implemented"
                 })

